### DISCUSSION TITLE
RuntimeError: cudnnGetConvolutionForwardAlgorithm not found.

### DISCUSSION THREAD

**Discussion by Ameng2126** [PaddlePaddle/PaddleOCR#13837] [Q&A]
-------------------------执行一下脚本-------------------------
import cv2
from paddleocr import PaddleOCR

def main():
    paddleocr = PaddleOCR(lang = 'ch',show_log=False,enable_mkldnn=True,
                          det_model_dir=r"./inference_model/det_model/Teacher",
                          rec_model_dir=r"./inference_model/rec_model/Teacher")

    img = cv2.imread(r"./output/imgTest/A027.png")

    result = paddleocr.ocr(img)

    for i in range(len(result[0])):
        print(result[0][i][1][0])

if __name__ == "__main__":
    main()
-------------------------出现一下报错-------------------------
E0906 20:25:01.906633 10400 port.cc:36] Load symbol cudnnGetConvolutionForwardAlgorithm failed.
Traceback (most recent call last):
  File "E:\Py\PaddleOCR-re\ocrDemo.py", line 19, in <module>
    main()
  File "E:\Py\PaddleOCR-re\ocrDemo.py", line 11, in main
    result = paddleocr.ocr(img)
  File "E:\Py\PaddleOCR-re\paddleocr.py", line 680, in ocr
    dt_boxes, rec_res, _ = self.__call__(img, cls)
  File "E:\Py\PaddleOCR-re\tools\infer\predict_system.py", line 76, in __call__
    dt_boxes, elapse = self.text_detector(img)
  File "E:\Py\PaddleOCR-re\tools\infer\predict_det.py", line 245, in __call__
    self.predictor.run()
RuntimeError: cudnnGetConvolutionForwardAlgorithm not found.

进程已结束,退出代码1

请问各位大佬这种问题怎么解决？

**Comment by GreatV** [comment]
按照安装指南重新安装一下paddle，注意cuda、cudnn版本对应 https://www.paddlepaddle.org.cn/install/quick?docurl=/documentation/docs/zh/install/pip/windows-pip.html
